### ISSUE TITLE
Crash on links

### ISSUE THREAD

**Issue by IlyaEremin** [smooch/smooch-android#43]
When I send `%[Button label here](http://anyurl.com)` or `%[Button label here](reply:PAYLOAD_HERE)` then app crashes with following message:
```
 java.lang.ArrayIndexOutOfBoundsException: length=16; index=17
at android.widget.RelativeLayout$LayoutParams.addRule(RelativeLayout.java:1204)
at io.smooch.ui.widget.MessageView.addSmoochButton(Unknown Source)
at io.smooch.ui.adapter.MessageListAdapter.addMessage(Unknown Source)
at io.smooch.ui.adapter.MessageListAdapter.getItemView(Unknown Source)
at io.smooch.ui.adapter.MessageListAdapter.getView(Unknown Source)
```
I'm using latest 4.0.3 sdk

**Comment by jugarrit** [comment]
This happens no matter what kind of button you send?
I can't reproduce this issue myself.

**Comment by jugarrit** [comment]
Does this occur when using the `ConversationFragment` outside of a `ConversationActivity`?

**Comment by jugarrit** [comment]
Seems to reproduce in Android 15-17.
I'll take a look

**Comment by IlyaEremin** [comment]
It occures both for `ConversationActivity` and `ConversationFragment`.

**Comment by jugarrit** [comment]
Fixed in 4.0.5
